packages/client/wallets/smart-wallet/src/error/index.ts

```diff
@@ -126,12 +127,26 @@ export class PasskeyPromptError extends SmartWalletSDKError {
     constructor(passkeyName: string) {
         super(
             `Prompt was either cancelled or timed out for passkey ${passkeyName}`,
+            undefined,
```

Comment:
seeing that we missed this in the previous PR, should this constructor take an object param instead of 3 separates?

Reply:
Good idea, I'll do it on top of these PRs